examples/demo/src/products/GridList.js

```diff
@@ -37,16 +41,16 @@ const GridList = ({ classes, ids, data, basePath }) => (
                         subtitle={
                             <span>
                                 {data[id].width}x{data[id].height},{' '}
-                                <b>
-                                    <NumberField
-                                        source="price"
-                                        record={data[id]}
-                                        options={{
-                                            style: 'currency',
-                                            currency: 'USD',
-                                        }}
-                                    />
-                                </b>
+                                <NumberField
+                                    className={classes.price}
```

Comment:
another side effect of the addition of `Typography` was that this component also lost its color